plugins/UM3NetworkPrinting/DiscoverUM3Action.py

```diff
@@ -25,24 +28,24 @@ def __init__(self):
 
         Application.getInstance().engineCreatedSignal.connect(self._createAdditionalComponentsView)
 
-        self._last_zeroconf_event_time = time.time()
-        self._zeroconf_change_grace_period = 0.25 # Time to wait after a zeroconf service change before allowing a zeroconf reset
+        self._last_zero_conf_event_time = time.time()
 
-    printersChanged = pyqtSignal()
+        # Time to wait after a zero-conf service change before allowing a zeroconf reset
+        self._zero_conf_change_grace_period = 0.25
```

Comment:
I can't see what the units are here. Is `_zero_conf_change_grace_period_seconds` an improvement?

Reply:
Same as last; splitting the code won't make it more reusable, but make changing it harder.

Reply:
I think your comment landed on the wrong part of the page here.